libs/arrow/build.gradle

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+// Notes:
+// - additional JVM arguments are added to distribution/src/config/jvm.options and ElasticsearchTestBasePlugin
+// - additional permissions are added to server/src/main/resources/org/elasticsearch/bootstrap/security.policy
+
+import org.elasticsearch.gradle.internal.precommit.CheckForbiddenApisTask
+
+apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.publish'
```

Comment:
```suggestion
```

I assume we do not wanna publish that as maven artifact. therefore we can remove `elasticsearch.publish`